tools/eventcontextcodegen/context_parser_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseEventContextFile(t *testing.T) {
+	r := require.New(t)
+
+	data, err := os.ReadFile("testdata/context.txt")
```

Comment:
What do you think about changing extension to `go`? Without it Github does not apply syntax highlighting to the file.

Reply:
This was on purpose, as i want to treat it as raw txt data. Otherwise your IDE/LSP will think that there are errors in the file.

Reply:
Understood.